src/vignette/storage/static_assets.clj

```diff
@@ -0,0 +1,48 @@
+(ns vignette.storage.static-assets
+  (:require [vignette.storage.protocols :refer :all]
+            [org.httpkit.client :as http]
```

Comment:
> Jet client only supports pure async interface. 
This should work for the static asset service-- as long as it returns the core.async channel back to jet for the [body](https://github.com/Wikia/vignette/pull/95/files#diff-5f65fe56082f716f52998b3d0642f4d3R20) think it should would work seamlessly.